data/od/testing/ifms/SSUDetails.json

```diff
@@ -0,0 +1,18 @@
+{
+    "tenantId": "od.testing",
+    "moduleName": "ifms",
+    "SSUDetails": [
+      {
+        "id": "1",
+        "ssuCode": "",
+        "ddoCode": "OLSHUD001",
+        "granteeAgCode": "GOHUDULBMPL0036",
```

Comment:
@Tulika-eGov Remove granteeAgCode, ddoCode, granteeName, ssuOffice from the config.